packages/instant/src/util/version.ts

```diff
@@ -0,0 +1,8 @@
+const anyWindow = window as any;
+
+export const versionUtil = {
+    writeVersionInfoToWindow: () => {
+        anyWindow.__zeroExInstantGitSha = process.env.GIT_SHA;
```

Comment:
Can we write this on the `zeroExInstant` window object? I feel like that'd be much cleaner.

Reply:
This is a great idea